nuget/lib/dependabot/nuget/file_parser/project_file_parser.rb

```diff
@@ -50,6 +55,63 @@ def dependency_set(project_file:)
 
         attr_reader :dependency_files
 
+        def add_sdk_references(doc, dependency_set, project_file)
+          # These come in 3 flavours:
+          # - <Project Sdk="Name/Version">
+          # - <Sdk Name="Name" Version="Version" />
+          # - <Import Project="..." Sdk="Name" Version="Version" />
+          # None of these support the use of properties, nor do they allow child
+          # elements instead of attributes.
+          add_sdk_refs_from_project(doc, dependency_set, project_file)
+          add_sdk_refs_from_sdk_tags(doc, dependency_set, project_file)
+          add_sdk_refs_from_import_tags(doc, dependency_set, project_file)
+        end
+
+        def add_sdk_ref_from_project(sdk_references, dependency_set, project_file)
+          sdk_references.split(";")&.each do |sdk_reference|
+            m = sdk_reference.match(PROJECT_SDK_REGEX)
+            if m
+              dependency = build_dependency(m[1], m[2], m[2], nil, project_file)
+              dependency_set << dependency if dependency
+            end
+          end
+        end
+
+        def add_sdk_refs_from_import_tags(doc, dependency_set, project_file)
+          doc.xpath("/Project/Import").each do |import_node|
+            next unless import_node.attribute("Sdk") && import_node.attribute("Version")
+
+            name = import_node.attribute("Sdk")&.value&.strip
+            version = import_node.attribute("Version")&.value&.strip
+
+            dependency =
+              build_dependency(name, version, version, nil, project_file)
+            dependency_set << dependency if dependency
+          end
+        end
+
+        def add_sdk_refs_from_project(doc, dependency_set, project_file)
+          doc.xpath("/Project").each do |project_node|
+            sdk_references = project_node.attribute("Sdk")&.value&.strip
+            next unless sdk_references
+
+            add_sdk_ref_from_project(sdk_references, dependency_set, project_file)
+          end
+        end
+
+        def add_sdk_refs_from_sdk_tags(doc, dependency_set, project_file)
+          doc.xpath("/Project/Sdk").each do |sdk_node|
+            next unless sdk_node.attribute("Version")
+
+            name = sdk_node.attribute("Name")&.value&.strip
+            version = sdk_node.attribute("Version")&.value&.strip
+
+            dependency =
+              build_dependency(name, version, version, nil, project_file)
```

Comment:
```suggestion
            dependency = build_dependency(name, version, version, nil, project_file)
```

We relaxed some linter rules a while ago. This is not important but I find it slightly easier on the eyes, feel free to ignore though.